src/sentry/static/sentry/app/components/sidebar/help.jsx

```diff
@@ -51,10 +51,10 @@ class SidebarHelp extends React.Component {
                 <SidebarMenuItem onClick={this.handleSearchClick}>
                   {t('Search Docs and FAQs')}
                 </SidebarMenuItem>
-                <SidebarMenuItem href="https://forum.sentry.io/" target="_blank">
+                <SidebarMenuItem href="https://forum.sentry.io/" openInANewTab>
                   {t('Community Discussions')}
                 </SidebarMenuItem>
-                <SidebarMenuItem href="https://status.sentry.io/" target="_blank">
+                <SidebarMenuItem href="https://status.sentry.io/" openInANewTab>
```

Comment:
`inNewTab` is a shorter way to say this. The `InANew` part looks awkward to me. :bike::house: